README.md

```diff
@@ -17,9 +17,10 @@ bower install bedrock-consumer
 Installation of the module followed by a restart of your [bedrock][]
 application is sufficient to make the module available.
 
-<!-- ## How It Works
+## API
 
-TODO: -->
+### handleCredential
 
+Process an Identity containing credentials by verifying the basic credential information and then emitting an event containing the verified credential information and all the data necessary to respond to the request. Credential consumer event listeners will determine how to consume the given credentials and then complete the response to the request via the res/next parameters.
```

Comment:
You should break at 75 characters for any text in README files like this. Human eyes get fatigued when they're required to scan left to right a lot (and on a large screen, lines w/o line breaks end up being really long).

Once you're done making this final change - squash your commit and let me know and I'll pull: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit